source/assets/stylesheets/index.scss

```diff
@@ -78,3 +78,17 @@
     }
   }
 }
+
+// This doesn't seem to be present -- it appears they removed it in one of the more recent releases.
+// Give me my rounded buttons, dammit!
+button.radius, .button.radius {
+    border-radius: 3px; }
+button.rounded, .button.rounded {
+  border-radius: 1000px; }
```

Comment:
Try to use units that are responsive to the font size, such as `em` or, for bonus points, `rem`. Designing to the pixel is going to make things harder to change in the future like raster versus vector graphics and having to change scale of an image.